man/duplicated.Rd

```diff
@@ -111,11 +112,17 @@ DT[,which.min(a)]            # row 10, the strictly smallest floating point valu
 identical(unique(DT),DT[1])  # TRUE, stable within tolerance
 identical(unique(DT),DT[10]) # FALSE
 
-# fromLast=TRUE
-DT <- data.table(A = rep(1:3, each=4), B = rep(1:4, each=3),
-                 C = rep(1:2, 6), key = c("A", "B"))
-duplicated(DT, by="B", fromLast=TRUE)
-unique(DT, by="B", fromLast=TRUE)
+# Explaining all usecases of fromLast in both 'duplicated' and 'unique' function.
```

Comment:
I'm not sure any change to the examples is warranted. I find the new examples too wordy; OP only requested we update the description of `fromLast`, I don't think we need to update the examples.

cc @tdhock do you think there's anything worth adding to the examples? I find the original clear & concise enough.

Reply:
Yeah if it's too wordy, we can remove them

Reply:
I think the unique example is helpful